docs/en/siem/installation.asciidoc

```diff
@@ -30,13 +32,77 @@ for sorting. For more information, see {kib}.
 [[install-beats]]
 === Install {beats} shippers
 
-To populate the security UI with metrics and
-log data, you need to install and configure the following shippers:
+To populate the {siem-ui} with hosts and network data, you need to install and
+configure Beats on the endpoints you want to collect and send data:
 
-* https://www.elastic.co/products/beats/packetbeat[{packetbeat}] for analyzing
-network packets 
 * https://www.elastic.co/products/beats/filebeat[{filebeat}] for forwarding and
 centralizing logs and files
 * https://www.elastic.co/products/beats/auditbeat[{auditbeat}] for monitoring
 directories for file changes
+* https://www.elastic.co/products/beats/winlogbeat[{winlogbeat}] for centralizing logs
+* https://www.elastic.co/products/beats/packetbeat[{packetbeat}] for analyzing
+network packets 
```

Comment:
I would use a more general wording "for analyzing network activity" here. "network packets" sounds more low level

Reply:
done - nice easy first fix